pandas/core/strings.py

```diff
@@ -1095,24 +1095,48 @@ def str_pad(arr, width, side='left', fillchar=' '):
 
 def str_split(arr, pat=None, n=None):
     """
+    Split strings around given separator/delimiter.
+
     Split each string (a la re.split) in the Series/Index by given
     pattern, propagating NA values. Equivalent to :meth:`str.split`.
 
     Parameters
     ----------
     pat : string, default None
-        String or regular expression to split on. If None, splits on whitespace
+        String or regular expression to split on. If None, split on whitespace.
```

Comment:
\`None\`

Reply:
@WillAyd small feedback: it's either no backticks or double backticks for small code snippets, single backticks are for parameter names. 
The question for such short things like None if we see them as 'code', but I think typically we do if it references a value that is passed.

(we should probably make a better overview of this in the docstring guidelines)